lgn-provers/src/provers/v1/query/mod.rs

```diff
@@ -12,6 +12,9 @@ pub(crate) mod dummy_prover;
 #[cfg(not(feature = "dummy-prover"))]
 pub(crate) mod euclid_prover;
 
+// TODO: what value?
+pub const ROW_TREE_MAX_DEPTH: usize = 20;
+pub const INDEX_TREE_MAX_DEPTH: usize = 20;
```

Comment:
* for row tree, if we want to support [USDC](https://etherscan.io/token/0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48), it's +2M entries so at least log should be 22.
* for index tree, if we want to 
   * support L1 with 12s block time for 2y it's at least 23
   * support Base with 2s block time for 1y it's at least 24.
 
So my recommendation would be row = 22, index = 24.
@nicholas-mainardi wdyt ?

Reply:
Now using MP2 constants, so we can move the discussion there.